test/test_models.py

```diff
@@ -36,18 +36,20 @@ def test_mfcc(self):
 class TestMelResNet(common_utils.TorchaudioTestCase):
 
     def test_waveform(self):
+        """test the output dimensions after _MelResNet block.
+        """
 
-        batch_size = 2
-        num_features = 200
-        input_dims = 100
-        output_dims = 128
-        res_blocks = 10
-        hidden_dims = 128
-        pad = 2
+        n_batch = 2
+        n_time = 200
+        n_freq = 100
+        n_output = 256
```

Comment:
why is the output shape changing?

Reply:
When I ran the test, I tried a different input value to double check if the test passes. Do you think I need to change it back to `n_output = 128`?

Reply:
Ok, thanks for clarifying. Let's change it back to what it was.

Reply:
Updated. Thanks.